salesagility/SuiteCRM
# Fixed #2266 - Email Template Editor

## Description



The email template editor has been modified to have a checkbox which brings up a textbox which allows users to enter the html directly without the editor modifying it.
## Motivation and Context

Address Issue #2266 - Access to core HTML for Emailtemplate existing and new 
The current editor modifies html in such a way that the formatting of many templates is lost when pasted in to the html source of it. 

## How To Test This

Create email template with html and see the html editor modify the template heavily. Test with fixes to see the html is intact(although it does go through the html purifier). After adding files please clear your browsers js cache. 
## Types of changes


### Final checklist
